features/FxA-83-sign-in-confirmation-email/README.md

```diff
@@ -267,12 +267,40 @@ sign-in confirmation.
 <img src="sign-in-success-time.png" height="300">
 
 This chart shows that most users complete sign-in
-confirmation within an hour. Unfortunately, this
-chart alone can not explain the difference between the other
-metrics.
+confirmation within an hour. This is important because if
+user's were taking greater than a day to confirm sign-in,
+it could skew metrics.
 
 The another thing that could account for the difference
 in these metrics is that one method might not be emitting
 or emitting twice, for different scenarios. The true
 sign-in confirmation success rate could possibly be a
 mix of these metrics.
+
+Something to note was that sign-in confirmation was enabled
+for 100% of users because of a security incident. Once the
+feature was fully enabled, the security incident stopped.
+Unfortunately, all of the metrics were not fully in
+place before the feature went to 100%, which makes it
+difficult to get a clear idea of what a baseline sign-in success
+rate looks like.
+
+The chart below shows that roughly,  less than 0.5% of our
+users click change password button from the sign-in confirmation
+email. Sign-in confirmation helps to provide these users
+with some protection on their accounts.
+
+<img src="sign-in-change-password.png" height="300">
```

Comment:
Viewing directly should work, https://github.com/mozilla/fxa/blob/e1c083df09cf7cc61f41e9033060c7bef5fd9a7f/features/FxA-83-sign-in-confirmation-email/README.md